src/pages/navbar/HomePage.tsx

```diff
@@ -1,22 +1,3 @@
-import { DialogsProvider, useDialogs } from '@toolpad/core/useDialogs';
-import Button from '@mui/material/Button';
-import CustomDialogWithDeviceConfiguration from '../../components/dialogs/CustomDialogWithDeviceConfiguration.tsx';
-
 export default function HomePage() {
-    const dialogs = useDialogs();
-
-    async function openDialog() {
-        await dialogs.open(CustomDialogWithDeviceConfiguration);
-    }
-
-    return (
-        <DialogsProvider>
-            <Button
-                onClick={() => {
-                    void openDialog();
-                }}>
-                Open custom
-            </Button>
-        </DialogsProvider>
-    );
+    return <></>;
```

Comment:
I think we can delete the homepage, it does not contain any code.

Reply:
I think for the moment we can keep `Home` because we still don`t know how our actual "Home" will look like. Maybe it is the smart-meters page, but maybe it is a completely new page and we would have to reimplement what i would delete here.